conformance/apis/v1alpha1/profilereport.go

```diff
@@ -23,26 +23,26 @@ type ProfileReport struct {
 	// (e.g. "AdminNetworkPolicy", "BaselineAdminNetworkPolicy")
 	Name string `json:"name"`
 
-	// Core indicates the core support level which includes the set of tests
+	// Standard indicates the standard support level which includes the set of tests
 	// which are the minimum the implementation must pass to be considered at
 	// all conformant.
-	Core Status `json:"core"`
+	Standard Status `json:"standard"`
 
-	// Extended indicates the extended support level which includes additional
+	// Experimental indicates the experimental support level which includes additional
 	// optional features which the implementation may choose to implement
 	// support for, but are not required.
```

Comment:
That's still describing "extended", not "experimental".

Reply:
nice catch, looks like I still missed some spots